src/main/java/io/orangebeard/listener/OrangebeardTestSystemListener.java

```diff
@@ -398,6 +388,26 @@ private StartTestItem getStartTestItem(TestPage testPage) {
         return testItem.build();
     }
 
+    private OrangebeardClient getOrangebeardClient() {
+        ApiVersion apiVersion = determineApiVersion(propertyFileName);
+        OrangebeardClient orangebeardClient;
```

Comment:
orangebeardClient hides this.orangebeardClient

Reply:
That's fine by me.

Reply:
Maybe this method could be made static. It depends on the question if `determineApiVersion` can be made static. (I've had a quick look at `determineApiVersion`, not sure if it can be made static).

Reply:
Yeah I looked at that; is too much of a bother I think.

Reply:
Then let's leave it for now.